tests/bud.bats

```diff
@@ -1640,13 +1641,40 @@ function _test_http() {
   run_buildah from ${target}
 }
 
-@test "bud-github-context-with-branch-and-subdir" {
+# Containerfile in this repo should only exists on older commit and
+# not on HEAD or the default branch.
+@test "bud-github-context-from-commit" {
+  if ! which git ; then
+    skip "no git in PATH"
+  fi
+  target=giturl-image
+  # Any repo would do, but this one is small, is FROM: scratch, local, and has
+  # its entire build context in a subdirectory of the repository.
+  if ! start_git_daemon ${TEST_SOURCES}/git-daemon/repo-with-containerfile-on-old-commit.tar.gz ; then
+    skip "error running git daemon"
+  fi
+  # Containerfile in this repo should only exists on older commit and
+  # not on HEAD or the default branch.
+  gitrepo=git://localhost:${GITPORT}/repo#f94193d34548eb58650a10a5183936d32c2d3280
+  run_buildah build $WITH_POLICY_JSON -t ${target} "${gitrepo}"
```

Comment:
Can we check for a specific error message here?

Reply:
Success build is enough to confirm that `Containerfile` was found and verifies the test case but I have added strings which should be part of build output as a double check.

Reply:
Ah, right, this one's expected to succeed.